docs/source/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+# sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('../../'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Hyperledger Cactus'
+copyright = '2021, Hyperledger Cactus'
```

Comment:
@AzaharaC Could this be a range instead where the start is 2020 and then the end of it is always the current year? (so right now it would be `2020-2021`)

https://www.delftstack.com/howto/python/python-current-year/

https://www.quora.com/Should-I-update-my-copyright-statement-to-the-current-year?share=1